typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumZircuitAmphrETHWarpConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  RouterConfig,
+  TokenRouterConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+const ethereumOwner = '0x008615770B588633265cB01Abd19740fAe67d0B9';
+const zircuitOwner = '0xD0673e7F3FB4037CA79F53d2d311D0e017d39963';
+const amphrEthCollateralAddress = '0x5fD13359Ba15A84B76f7F87568309040176167cd';
+
+export const getEthereumZircuitAmphrETHWarpConfig = async (
+  routerConfig: ChainMap<RouterConfig>,
+): Promise<ChainMap<TokenRouterConfig>> => {
+  const ethereum: TokenRouterConfig = {
+    ...routerConfig.ethereum,
+    type: TokenType.collateral,
+    token: amphrEthCollateralAddress,
```

Comment:
Can we add the token address here please? https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/6dae7f9f5562bb5d2c140aeee5ef382fe9f501c9/typescript/infra/src/config/warp.ts#L5